src/components/TezosOperationView.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { getOperation } from '../Conseil';
+
+interface TezosOperationProps {
+    network: string;
+    id: string;
+}
+
+interface TezosOperationState {
+    data: object;
+}
+
+export class TezosOperationView extends React.Component<TezosOperationProps, TezosOperationState> {
+
+    constructor(props: TezosOperationProps) {
+        super(props);
+        this.state = {data: {}};
+        this.refreshData = this.refreshData.bind(this);
+        this.refreshData(props);
+    }
+
+    refreshData(props: TezosOperationProps) {
+        getOperation(this.props.network, this.props.id).
```

Comment:
From what I've seen, usually the `.` stays with the `then`:
```
getOperation(this.props.network, this.props.id)
.then(value => this.setState({...
```

Reply:
[async await](https://javascript.info/async-await) is what all the cool kids are doing now.
```
refreshData = async (props: TezosOperationProps) {
  const value = await getOperation(this.props.network, this.props.id);
  this.setState({ data: JSON.parse(decodeURI(value) })
}
```

Reply:
Changed the dot.

Reply:
Async/await added as issue #22.